### ISSUE TITLE
Text not visible under linux while using dark themes

### ISSUE THREAD

**Issue by MinerMax555** [richardwilkes/gcs#88]
While using a dark theme in Gnome (Ubuntu 18.04), both the background and text in most menus is white, making it unreadable. The problem comes from GCS changing the font color according to the theme, but not the background color

**Comment by StalenBas** [comment]
A workaround I found is to launch gcs with the environment variable `GTK_THEME=theme:variant`

See [here](https://unix.stackexchange.com/questions/14129/gtk-enable-set-dark-theme-on-a-per-application-basis) for instructions.

**Comment by richardwilkes** [comment]
Yeah... themes on all platforms are a nightmare to support, as the APIs necessary to even notice them from Java are often non-existent.

I'm going to rename this to essentially request that all colors used in the software be settable via preferences... not as good as following the theme you've set, but lets you manually tweak things to effectively follow it.

**Comment by crnormand** [comment]
I could certainly look into this, now that we have a "System" preferences page.    Can you give me some examples of where GCS is setting the colors?   Or is it just spread throughout?

**Comment by richardwilkes** [comment]
If this was implemented, I'd want it in a separate "Colors" or maybe "Theme" tab. As for where the colors are... yeah, pretty much scattered all over the place, including some in the toolkit, which makes it a pain.

**Comment by crnormand** [comment]
Certainly.   At least I now know how to make a new sheet.   I guess I can go search for references to the Color class,,, any other suggestions for finding them?

**Comment by richardwilkes** [comment]
Pretty much just searching for uses of Color and (I think) SystemColor.
